src/main/java/uk/gov/justice/digital/hmpps/whereabouts/model/AttendanceChangeValues.kt

```diff
@@ -6,7 +6,7 @@ enum class AttendanceChangeValues {
   SessionCancelled,
   RestInCellOrSick,
   RestDay,
-  UnacceptableAbsence,
+  UnacceptableAbsenceIncentiveLevelWarning,
```

Comment:
really don't understand why this is a separate enum, but have now added a test to ensure that they match